common/nns.go

```diff
@@ -46,7 +45,12 @@ func ResolveFSContractWithNNS(nns interop.Hash160, contractName string) interop.
 		panic("did not find a record of the " + contractName + " contract in the NNS")
 	}
 	if len(records[0]) == 2*interop.Hash160Len {
-		return convert.ToBytes(std.Atoi(records[0], 16))
+		var h = make([]byte, interop.Hash160Len)
+		for i := 0; i < interop.Hash160Len; i++ {
+			ii := (interop.Hash160Len - i - 1) * 2
+			h[i] = byte(std.Atoi(records[0][ii:ii+2], 16))
```

Comment:
magic to me but lets pretend it is OK

Reply:
hex byte -> byte, one by one with reversed order